Modules/TPC/include/TPC/CheckOfTrendings.h

```diff
@@ -41,6 +41,17 @@ class CheckOfTrendings : public o2::quality_control::checker::CheckInterface
 
  private:
   ClassDefOverride(CheckOfTrendings, 2);
+  std::string mCheckChoice;
+  float mExpectedPhysicsValue;
+  float mNSigmaExpectedPhysicsValue;
+  float mNSigmaBadExpectedPhysicsValue;
+  float mNSigmaMean;
+  float mNSigmaBadMean;
+  int mPointToTakeForExpectedValueCheck;
+  int mPointToTakeForMeanCheck;
+  static constexpr std::string_view mCheckChoiceMean = "Mean";
+  static constexpr std::string_view mCheckChoiceExpectedPhysicsValue = "ExpectedPhysicsValue";
+  static constexpr std::string_view mCheckChoiceBoth = "Both";
```

Comment:
```suggestion
  static constexpr std::string_view CheckChoiceMean = "Mean";
  static constexpr std::string_view CheckChoiceExpectedPhysicsValue = "ExpectedPhysicsValue";
  static constexpr std::string_view CheckChoiceBoth = "Both";
```
See naming convention for `constexpr` https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Variable_Names#Variable_Names